kolibri/plugins/learn/assets/src/views/LibraryPage/index.vue

```diff
@@ -416,7 +414,7 @@
       };
     },
     computed: {
-      ...mapGetters(['isLearnerOnlyImport']),
+      ...mapGetters(['isLearner']),
```

Comment:
My only question is about this change - I think what we probably want to do here is to retain the `isLearnerOnlyImport` condition, _and_ also add a condition checking if the user is a superadmin, otherwise, for the first imported learner only import user, the modal will now still display.